cfme/tests/cli/test_appliance_console.py

```diff
@@ -760,7 +759,30 @@ def test_appliance_console_restart():
             9.
             10. Confirm that Appliance is running into browser.
     """
-    pass
+    cmd_stop_EVM = ("ap", RETURN, "16", "Y", RETURN, RETURN, "21")
+    appliance.appliance_console.run_commands(cmd_stop_EVM, timeout=90)
+    # Checking EVM service stoppped or not
+    wait_for(
+        lambda: not appliance.is_web_ui_running(),
```

Comment:
I don't think you should be using `lambda` here since you're calling a function.
You can simply do `wait_for(appliance.is_web_ui_running, fail_condition=True, ...)`.
Using `fail_condition=True` here is equivalent to using `not appliance.is_web_ui_running()`.

Reply:
Getting below error if I do not use lambda.
```
        else:
>           f_code = func.__code__
E           AttributeError: 'bool' object has no attribute '__code__'

```

Reply:
Will change `assignee` shortly